cli/admin/src/main/java/io/pravega/cli/admin/utils/CLIConfig.java

```diff
@@ -91,7 +91,7 @@
      * Truststore access token ttl in seconds if TLS is configured in the CLI.
      */
     @Getter
-    private final Duration accessTokenTtlInSeconds;
+    private final Duration accessTokenTtl;
```

Comment:
This name is fine. This was the only change I asked you to do with my previous review's comment.